model/pod/podutil.go

```diff
@@ -0,0 +1,117 @@
+package pod
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/pkg/errors"
+)
+
+const (
+	executable = "./evergreen"
```

Comment:
Rather than define a constant for this, I would just use
```go
fmt.Sprintf("./%s, p.BinaryPath())
```
on L45 since the entire agent command download/execution script is dependent on all the commands being relative to the working directory. Making assumptions about relative/absolute paths is generally risky as it's a common source of bugs (e.g. doing something in a relative path when the "working directory" isn't exactly what you expect it to be). Therefore, I would try to localize relative path dependencies like this to be as close as possible to other dependencies on relative paths `curl -LO` and `chmod ./evergreen` depend on relative paths, so the dependency on invoking the evergreen binary in a relative path should also be localized to that spot rather than be defined as a constant visible to the entire package.